app/helpers/Intersect.js

```diff
@@ -402,3 +432,21 @@ export function disableDebugMode() {
   // Kick off intersection calculations to restore data
   checkIntersect();
 }
+
+/**
+ * Records where user has crossed paths to avoid repeated notifications or checking
+ *
+ * @param {array} localArray - Array of user locations
+ * @param {number} i - Index of current location in the localArray
+ * @param {array} infectedLocation - Array of infected locations
+ * @param {number} newIntersections - Counter of new intersections found
+ */
+function addNewIntersection(localArray, i, infectedLocation, newIntersections) {
+  console.log('New intersection');
+
+  if ('crossedPaths' in localArray[i])
+    localArray[i]['crossedPaths'].push(infectedLocation);
```

Comment:
Mutating an object that you don't own (didn't create) is asking for trouble. 

I'd really like to avoid mutating `localArray`, but if that was the intended behaviour by @kenpugsley that localArray would progressively be mutated, I'll allow it. @kenpugsley  can you please weigh-in?

It can often end up in hard to debug state issue later.